libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/Flyout.tsx

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  Icon,
+  Image,
+  ImageFit,
+  List,
+  Panel,
+  PanelType,
+  FocusZone,
+  Stack,
+  Text,
+  Separator
+} from "@fluentui/react";
+import { localization } from "@responsible-ai/localization";
+import React from "react";
+
+import { IDatasetSummary } from "../Interfaces/IExplanationDashboardProps";
+
+import { flyoutStyles } from "./Flyout.styles";
+import { IListItem } from "./ImageList";
+
+export interface IFlyoutProps {
+  data: IDatasetSummary;
+  isOpen: boolean;
+  item: IListItem | undefined;
+  callback: () => void;
+}
+
+export class IFlyoutState {}
+
+const stackTokens = {
+  childrenGap: "l1"
+};
+
+export class Flyout extends React.Component<IFlyoutProps, IFlyoutState> {
+  public constructor(props: IFlyoutProps) {
+    super(props);
+    this.state = {};
+  }
+
+  public render(): React.ReactNode {
+    const classNames = flyoutStyles();
+
+    const item = this.props.item;
+
+    const list = [];
+
+    for (let i = 0; i < 30; i++) {
+      list.push({ title: "Feature 1", value: "value" });
```

Comment:
what is this? is it fake data?

Reply:
Yes it's fake data until @jameschao hooks up the backend to pass the metadata features for images

Reply:
@imatiach-msft - I think you mean @jamesbchao and not me @jameschao. A great name nonetheless!

Reply:
oh sorry! yes I meant @jamesbchao :)